Dockerfile.rhel7

```diff
@@ -1,7 +1,7 @@
 FROM registry.svc.ci.openshift.org/ocp/builder:golang-1.10 AS builder
 WORKDIR /go/src/github.com/openshift/console-operator
 COPY . .
-RUN make build
+RUN ADDITIONAL_GOTAGS="ocp" make build WHAT="cmd/console"
```

Comment:
So the `Dockerfile` does not need to be updated due to the `build tag` below `+build !ocp` that will include `brand_okd.go` by default.

Reply:
correct